backend/census_historical_migration/sac_general_lib/sac_creator.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+from django.apps import apps
+
+from census_historical_migration.sac_general_lib.general_information import (
+    _general_information,
+)
+from census_historical_migration.sac_general_lib.audit_information import (
+    _audit_information,
+)
+from census_historical_migration.sac_general_lib.auditee_certification import (
+    _auditee_certification,
+)
+from census_historical_migration.sac_general_lib.auditor_certification import (
+    _auditor_certification,
+)
+from census_historical_migration.sac_general_lib.report_id_generator import (
+    dbkey_to_report_id,
+)
+
+from census_historical_migration.workbooklib.census_models.census import (
+    CensusGen22 as Gen,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def _create_sac(user, dbkey):
+    """Create a SAC object for the historic data migration."""
+    SingleAuditChecklist = apps.get_model("audit.SingleAuditChecklist")
+    generated_report_id = dbkey_to_report_id(Gen, dbkey)
+    try:
+        exists = SingleAuditChecklist.objects.get(report_id=generated_report_id)
+    except SingleAuditChecklist.DoesNotExist:
+        exists = None
+    if exists:
+        exists.delete()
+
+    sac = SingleAuditChecklist.objects.create(
+        submitted_by=user,
+        general_information=_general_information(dbkey),
+        audit_information=_audit_information(dbkey),
+    )
+
+    sac.report_id = generated_report_id
+
+    Access = apps.get_model("audit.Access")
+    Access.objects.create(
+        sac=sac,
+        user=user,
+        email=user.email,
+        role="editor",
+    )
+
+    # We need these to be different.
+    Access.objects.create(
+        sac=sac,
+        user=user,
+        email="bob_the_auditee_official@auditee.org",  # user.email, FIXME - Confirm this is the right value?

```

Comment:
Like other values, we might need this to be more descriptive? Not sure what we should do here.

Actually, I do.

For now, I think we clean it up a bit. However, this needs a ticket and some discussion.

We're going to either want to insert a clearly defined placeholder, or we're going to want to pull values from the General form for the auditor and auditee contact, but... perhaps label them as choices we made.

What I'm concerned about is that these are _access objects_, and therefore they'll end up in our system. We're creating them because they get accessed as part of the migration. 

Hence... using one set of "fake" access objects makes sense.

But, they should still be `fac-census-migration-auditee-official` or similar.

Reply:
Using `fac-census-migration-auditee-official@auditee.org` for now but also added this to the [ticket](https://github.com/GSA-TTS/FAC/issues/2832)